scala-base/1.0/Dockerfile

```diff
@@ -0,0 +1,51 @@
+FROM quay.io/continuouspipe/ubuntu:16.04_v1
+
+MAINTAINER Sarunas Valaskevicius <svalaskevicius+cp-dockerfiles@inviqa.com>
+
+# based on java:8 and https://github.com/hseeberger/scala-sbt images
+
+ENV SBT_VERSION 0.13.13
+
+ENV JAVA_HOME /usr/lib/jvm/java-8-openjdk-amd64
+ENV JAVA_VERSION 8u111
+ENV JAVA_DEBIAN_VERSION 8u111-b14-2ubuntu0.16.04.2
```

Comment:
why hard-code the version in? we want security updates to happen as part of release cycle